chapter_appendix_math/geometry-linear-algebric-ops_vn.md

```diff
@@ -67,7 +78,21 @@ However, it can be beneficial to be flexible.
 Matrices are useful data structures: they allow us to organize data that have different modalities of variation. For example, rows in our matrix might correspond to different houses (data points), while columns might correspond to different attributes. This should sound familiar if you have ever used spreadsheet software or have read :numref:`sec_pandas`. Thus, although the default orientation of a single vector is a column vector, in a matrix that represents a tabular dataset, it is more conventional to treat each data point as a row vector in the matrix. And, as we will see in later chapters, this convention will enable common deep learning practices. For example, along the outermost axis of an `ndarray`, we can access or enumerate minibatches of data points, or just data points if no minibatch exists.
 -->
 
-*dịch đoạn phía trên*
+Những biểu diễn này thường có những cách diễn giải khác nhau. Các điểm dữ liệu
+được biểu diễn bằng các vector cột và các trọng số dùng trong các tổng có
```

Comment:
câu này khó dịch cho ra nghĩa nếu không có liên hệ với thuật ngữ tiếng Anh. Mình tạm đề nghị như sau

.. và các trọng số (tiếng Anh: weight vector) - để sử dụng trong tính toán "weighted sum" (là phép tính cộng của nhiều thành phần, sau khi nhân mỗi thành phần với trọng số tương ứng) - được biểu diễn bằng vector hàng.